### PR TITLE
Trying to make the CourseDetailScreen utilize course objects. Stuck at reading a mock XML from a file

### LINKED ISSUE #22: make CourseDetailScreen functional

### PR DESCRIPTION
**Pull request by michalrzak**

### CODE REVIEW

**Comment by memoriesadrift** [comment]
Pushed a fix - we can await the result of a simple function call that returns the xml string.
Edit: See https://api.flutter.dev/flutter/dart-async/Future-class.html

**Comment by peter-gy** [comment]
The following lines would solve the issue:
```
import 'dart:io'
import 'package:xml/xml.dart';

final xmlFile = File('course.xml');
final xmlString = xmlFile.readAsStringSync();
final document = XmlDocument.parse(xmlString);
```

**Comment by memoriesadrift** [comment]
The thing is, i assume, that we want to mock the async nature of the api requests so that we can use `FutureBuilder`, meaning a Future is required, so sync functions don't adequately solve it.

**Comment by michalrzak** [comment]
> The following lines would solve the issue:
> 
> ```
> import 'dart:io'
> import 'package:xml/xml.dart';
> 
> final xmlFile = File('course.xml');
> final xmlString = xmlFile.readAsStringSync();
> final document = XmlDocument.parse(xmlString);
> ```

it's working now and its only a temporary solution anyway so let's just keep it imo

**Comment by michalrzak** [comment]
Please somebody pull this and run it and check out what else you would like to see in the course screen. 
The design is trivial to change, right now I am more after what other information to display there.

File: lib/screens/CourseDetailScreen.dart [hunk 1]
```diff
@@ -1,34 +1,30 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_html/flutter_html.dart';
 import 'package:we_find/model/I18NString.dart';
```
Comment (memoriesadrift) [inline lib/screens/CourseDetailScreen.dart]: Currently this is unused, shall we create a separate issue for implementing multilingual support?

File: lib/widgets/course_detail_widgets.dart [hunk 2]
```diff
@@ -21,7 +22,7 @@ class _GroupPickerState extends State<GroupPicker> {
     // which this would not detect
     for (int i = 0; i < widget._groups.length; ++i) {
       items.add(DropdownMenuItem(
-        child: Text((i + 1).toString()),
+        child: Text(widget._groups[i].getId()),
```
Comment (memoriesadrift) [inline lib/widgets/course_detail_widgets.dart]: Shouldn't we display both the group number and the ID? I know the id is encoded in the group but it's bad UX for new users who don't know, right?

File: lib/widgets/course_detail_widgets.dart [hunk 3]
```diff
@@ -82,10 +84,19 @@ class GroupDetails extends StatelessWidget {
...
-    if (_group.schedule != null) {
-      view.add(EventCalendar(_group.schedule!));
+    try {
+      view.add(EventCalendar(_group.getSchedule()));
+    } catch (e) {
+      // do nothing
     }
 
+    // Minimum requirements
+    view.add(Text(
```
Comment (memoriesadrift) [inline lib/widgets/course_detail_widgets.dart]: Let's add padding around these elements and increase font size!
I find it way too small to read and also very bunched up together, it looks too dense. 
Also, to reconsider is the font colour, I feel like it's a little bit too light.

File: lib/widgets/course_detail_widgets.dart [hunk 4]
```diff
@@ -105,39 +116,27 @@ class GroupDetails extends StatelessWidget {
...
+        Text(event.getRoom())
       ],
     );
   }
 
   Column _buildEventCalendar() {
     List<Row> view = [];
 
-    for (Event event in _schedule.events) {
+    for (EventTranslate event in _schedule.getEvents()) {
```
Comment (memoriesadrift) [inline lib/widgets/course_detail_widgets.dart:155]: Would love to get an overview of how this works, ping me when you have time so we can talk about it

**Comment by memoriesadrift** [comment]
When ready, click ready for review and I'll run through it again.

**Pull request merged**